src/HidLibrary/unit/Tuple_VidPid.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace HidLibrary.unit
+{
+    public class Tuple_VidPid
```

Comment:
This is a really unwieldy name for this class, unless it adheres to a new C# convention I don't know about. It's not actually a tuple, since it's just a regular C# data object. It seems like something along these lines *could* be useful, but it should probably include a UsageID property and be called something like `HidDeviceDescription`, and we should use it inside `HidDevice` as well, possibly as the underlying data storage for the properties it describes.